app/code/Magento/InventorySales/Test/Integration/Stock/Validator/WebsiteAssignedToStockValidatorTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\InventoryApi\Api\StockRepositoryInterface;
+use Magento\InventoryApi\Model\StockValidatorInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Provides tests for stock validator.
+ */
+class WebsiteAssignedToStockValidatorTest extends TestCase
```

Comment:
Looks like we need to remove this class due to we have this check in api-functional tests